data/human/free worlds 0 prologue.txt

```diff
@@ -944,7 +942,7 @@ mission "FW Escort 3"
 			`When you walk into the bar, Tomek looks very glad to see you. "Here's the deal," he says. "One of our freighters, the <npc>, was just disabled in the <waypoints> system. We need you to fly there immediately, repair it, and bring it back here before it can be captured by pirates. Do you think you're up to the task?"`
 			choice
 				`	"Sure!"`
-				`	"Sorry, I'm just a merchant captain. The Free Worlds shouldn't be asking me to do combat missions."`
+				`	"Escort duty was one thing, but now you're ordering me around like you're my superior? I'm out."`
```

Comment:
This seems overly aggressive. Tomek is offering you a job, and I don't think it reads like he's giving you a direct order.
```suggestion
				`	"Sorry, no. I'm just a merchant captain. Find someone else to do combat missions."`
```

Reply:
It's also not a combat mission though. Tomek's brief doesn't say you'll have to fight off the pirates to save the Shackleton, just that you have to repair it and escape (which is usually what happens in practice). There's more combat in the previous two escort jobs than in this mission, so why would the player character turn down this mission for "combat" reasons at this point?

Reply:
Whoops, that's a good point. Still not sure the superior thing works though...

Maybe:
```suggestion
				`	"Sorry, no. I'm a merchant captain with bills to pay. I'm not going to go search for your missing ships in the middle of nowhere."`
```